packages/tailwindcss/src/utilities.test.ts

```diff
@@ -15071,6 +15066,11 @@ test('text', async () => {
       line-height: calc(var(--spacing) * 6);
     }
 
+    .text-sm {
+      font-size: var(--text-sm);
+      line-height: var(--tw-leading, var(--text-sm--line-height));
+    }
+
```

Comment:
This now uses alphabetical sorting becuase it has the same number of props as the arbitrary text classes